src/opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -237,16 +237,9 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 13;
 
-  // A highly recommended but not required flag that identifies when a
-  // trace crosses a process boundary. True when the parent_span belongs
-  // to the same process as the current span. This flag is most commonly
-  // used to indicate the need to adjust time as clocks in different
-  // processes may not be synchronized.
-  google.protobuf.BoolValue same_process_as_parent_span = 14;
-
   // An optional number of child spans that were generated while this span
   // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 15;
+  google.protobuf.UInt32Value child_span_count = 14;
```

Comment:
At the top of the Span proto message there is a message with the next id, please update.

Reply:
Good catch, fixed.